src/rendering/renderers/webGpu/WebGpuRenderer.js

```diff
@@ -313,7 +313,8 @@ export class WebGpuRenderer extends Renderer {
 			if (!light.entity) continue;
 			this.lightsBuffer.appendData(light.entity.pos);
 			this.lightsBuffer.skipBytes(4);
-			this.lightsBuffer.appendData(light.color);
+			// FIXME: a bit hacky; using color.multiplyScalar(intensity) doesn't work since the light.color vector will reclamp itself between 0 and 1.
+			this.lightsBuffer.appendData(new Vec3(light.color.x * light.intensity, light.color.y * light.intensity, light.color.z * light.intensity));
```

Comment:
I think you might be able to do `light.color.clone().multiply(intensity)` (or `multiplyScalar`). I think the issue is that otherwise it modifies light.color in place and it will end up multiplying in an infinite loop.

Reply:
Good eye! I realized the issue was due to `light.color` clamping itself, but never caught the `clone()` method available for use.